README.md

```diff
@@ -1,2 +1,31 @@
 # aws-synthetic-checks
+
 AWS re-usable synthetic uptime &amp; graphql operation checks stored in S3
+
+## Local Development
+
+### Fresh Setup
+
+Clone the repo:
+
+-   `git clone git@github.com:Pocket/aws-synthetic-checks.git`
+-   `cd aws-synthetic-check`
+
+To compile AWS infra:
+
+-   `cd .aws`
+-   `npm ci`
+-   `npm run build:dev`
+
+To build and create zip file with synthetic code:
+
+-   `cd aws-synthetic-check`
+-   `npm ci`
+-   `npm run build:local`
+
+The created zip file `aws-synthetic.zip` will be located in `tmp-dist/synthetics-export`
+
+To run canary code locally:
```

Comment:
to do: add how to use the local wrapper.

Reply:
Yep, I will follow up in another PR once I get it working locally